src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -12,6 +10,7 @@ public static class ClaimType
             public const string AzureSignalRSysPrefix = "asrs.s.";
             public const string AuthenticationType = AzureSignalRSysPrefix + "aut";
             public const string UserId = AzureSignalRSysPrefix + "uid";
+            public const string OriginalPath = AzureSignalRSysPrefix + "op";
```

Comment:
Yeah, thought about it already.
The difference is the original request path have to be passed from app server to service, because it will be lost when client is redirected from app server to service.
In my approach, I put it as one of the system claims. Since claims are already passed back to app server, I think there is no need to add an extra property to the `OpenConnectionMessage`.

Reply:
Another benefit is that we don't have change the service runtime.